src/Caching/Hybrid/src/Runtime/HybridCacheEntryOptions.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Extensions.Caching.Hybrid;
+
+/// <summary>
+/// Additional options (expiration, etc) that apply to a <see cref="HybridCache"/> operation. When options
+/// can be specified at miltiple levels (for example globally and per-call), the values are composed; the
+/// most granular non-null value is used, with null values being inherited. If no value is specified at
+/// any level, the implementation may choose a reasonable default.
+/// </summary>
+public sealed class HybridCacheEntryOptions
+{
+    /// <summary>
+    /// Overall cache duration of this entry, passed to the backend distributed cache
+    /// </summary>
+    public TimeSpan? Expiration { get; init; } // overall cache duration
+
+    /// <summary>
+    /// Cache duration in local cache; when retrieving a cached value
+    /// from an external cache store, this value will be used to calculate the local
```

Comment:
I think this comment is saying that we're going to adjust the value you specify if it exceeds `Expiration` (and possibly under other circumstances?)?